### PR TITLE
Update Source fields, Added unit translations

### PR DESCRIPTION
**Pull request by isaaccodekill**
- Update the source fields to the new Data source translation. 
- Added translations for unit measurements

### CODE REVIEW

File: app/src/components/Modals/activity-modal/activity-modal-body.tsx [hunk 1]
```diff
@@ -103,6 +103,14 @@ const ActivityModalBody = ({
     }
   };
 
+  const filteredfields = fields.filter((f) => {
```
Comment (lemilonkh) [inline app/src/components/Modals/activity-modal/activity-modal-body.tsx:106]:
```suggestion
  const filteredFields = fields.filter((f) => {
```

File: app/src/components/Modals/activity-modal/activity-modal-body.tsx [hunk 2]
```diff
@@ -659,86 +667,143 @@ const ActivityModalBody = ({
...
+                />
+              </InputGroup>
+              {(errors?.activity?.[sourceField.id] as any) ? (
+                <Box display="flex" gap="6px" alignItems="center" mt="6px">
+                  <WarningIcon color="sentiment.negativeDefault" />
+                  <Text fontSize="body.md">
+                    {" "}
+                    {errors?.activity?.[sourceField.id]?.message}{" "}
+                  </Text>
+                  {/* use ii8n */}
```
Comment (lemilonkh) [inline app/src/components/Modals/activity-modal/activity-modal-body.tsx:754]: Add TODO and ticket?

**Review by lemilonkh — APPROVED** [review]
Thanks ✨

File: app/src/components/Modals/activity-modal/activity-modal-body.tsx (hunk 2 above)
Comment (isaaccodekill) [reply app/src/components/Modals/activity-modal/activity-modal-body.tsx:754]: this is an old comment

**Pull request merged**